app/my_jss/accounts/AccountService.js

```diff
@@ -0,0 +1,41 @@
+
+
+function AccountService() {
+
+};
+
+AccountService.prototype.signUp = function(name,email,password){
+  var bool = false;
+
+  if(!usersClass.checkedByEmail(email)){
+        usersClass.addNewUser(name,email,password);
```

Comment:
Put a return false statement here,. remove else { branch and replace it with a final return statement